lib/features/guide/widgets/about_us/models/about_us_details.dart

```diff
@@ -0,0 +1,34 @@
+import '../../../../../api_base/directus_assets_url.dart';
+import '../../../../../utils/url_icons.dart';
+import '../../../../../utils/where_non_null_iterable.dart';
+import '../repository/about_us_repository.dart';
+import 'member_data.dart';
+
+class AboutUsDetails {
+  final AboutUs? aboutUs;
+  final List<AboutUsTeam>? aboutUsTeam;
+
+  AboutUsDetails({this.aboutUs, this.aboutUsTeam});
+
+  List<MemberData> getMemberData(List<AboutUsTeam>? aboutUsTeam) {
+    return aboutUsTeam?.whereNonNull.map((e) {
+          return MemberData(
+            name: e.name,
+            imageUrl: e.photo?.filename_disk?.directusUrl,
+            socialLinks:
+                e.socialLinks?.whereNonNull.map((e) => e.url).toList() ?? [],
+            subtitle: e.subtitle,
+          );
+        }).toList() ??
+        [];
+  }
+
+  List<UrlIconsUtilModel> getSocialIcons(AboutUs? aboutUs) {
```

Comment:
same here?

Reply:
I tried but only to do it is something like this
`TeamSection(
                    members: value?.getMemberData() ?? [],
                  ),`
and function:
`List<MemberData> getMemberData() {
    return aboutUsTeam.whereNonNull.map((e) {
      return MemberData(
        name: e.name,
        imageUrl: e.photo?.filename_disk?.directusUrl,
        socialLinks:
            e.socialLinks.whereNonNull.map((e) => e.url).toList(),
        subtitle: e.subtitle,
      );
    }).toList();
  }`
  
  Is this what you were thinking about?